Main.cs

```diff
@@ -139,32 +120,50 @@ private void GenerateThumb()
             }
 
             // 1) some setup
-            var tmpImage = (Image)cachedImage.Clone();
+            var tmpImage = new Bitmap(cachedImage.Width, cachedImage.Height);
+            var copyImage = (Image)cachedImage.Clone();
+
+            var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42));

```

Comment:
i think both newsolidbrush and pen are IDisposable..

if so please use

using (var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42))
using (var backgroundEdgeColor = new Pen(Color.FromArgb(42, 42, 42)))
{
}

Reply:
do the same for any IDisposable object please